clang/include/clang/Driver/Options.td

```diff
@@ -1886,6 +1886,11 @@ def fprofile_update_EQ : Joined<["-"], "fprofile-update=">,
     Values<"atomic,prefer-atomic,single">,
     MetaVarName<"<method>">, HelpText<"Set update method of profile counters">,
     MarshallingInfoFlag<CodeGenOpts<"AtomicProfileUpdate">>;
+def fprofile_continuous : Flag<["-"], "fprofile-continuous">,
```

Comment:
Mild suggestion: `-fprofile-continuously` reads a little better. But happy with the existing flag if you feel strongly about it.

Reply:
Thanks for the feedback. I prefer `-fprofile-continuous` since everywhere we call this feature the "continuous mode". If others feel strongly about changing it, then I can ofcourse change it.